docs/sdk/quickstart-by-language/quickstart-nodejs.mdx

```diff
@@ -0,0 +1,179 @@
+their applications. This SDK simplifies the implementation of fine-grained access controls, whether you're managing user roles with
+[RBAC](/how-to/build-policies/rbac/overview), policies with [ABAC](/how-to/build-policies/abac/overview), relationships with
+[ReBAC](/how-to/build-policies/rebac/overview), or making use of other powerful Permit features like [Elements](/embeddable-uis/overview).
+
+:::note Prerequisites:
+This tutorial is for users who have already set up at least one policy in Permit.io. If you haven't done so yet,
+please refer to [this guide](/quickstart) first.
+:::
+
+## Adding the Permit.io Node SDK to your project
+
+### 1. Install the Permit.io SDK
+
+To start, you need to install the Permit.io SDK in your project. Open your terminal and run the following command:
+
+```sh
+npm install permitio
+```
+
+This will add the Permit.io SDK to your project, allowing you to integrate authorization management seamlessly.
+
+### 2. Import the SDK into your code
+
+After installing the SDK, you need to import it into your application.
+
+```javascript
+import { Permit } from "permitio";
+```
+
+### 3. Create a new Instance of the SDK
+
+Next, create a new Permit object instance of the Permit SDK. You'll need a secret API key, which you can obtain by following the instructions
+provided in the setup section of the Permit.io documentation.
+
+```javascript
+// This first line initializes the SDK and connects your Node.js app
+// to the Permit.io PDP container you've set up in the previous step.
+const permit = new Permit({
+  // your API Key
+  token: "[YOUR_API_KEY]",
+
+  // in production, you might need to change this url to fit your deployment
+  pdp: "http://localhost:7766",
+
+  // if you want the SDK to emit logs, uncomment this:
+  // log: {
+  //   level: "debug",
+  // },
+
+  // The SDK returns false if you get a timeout / network error
+  // if you want it to throw an error instead, and let you handle this, uncomment this:
+  // throwOnError: true,
+});
+```
+
+Replace **`YOUR_API_KEY`** with the actual API key you obtained. Find instructions how to fetch your API
+key [here](/overview/connecting-your-app#1-get-your-permit-environment-api-key).
+
+## The SDK enforcement functionality - Checking for permissions
+
+`permit.check()` is the fundamental SDK function that controls the permissions enforcement in your application code. It acts as a guard of who
+can do what, based on the policy that you defined.
+
+:::tip
+If you would like to learn more about the `permit.check()` function, and how it works with different policy models, you can read
+more about it [here](/how-to/enforce-permissions/check).
+:::
+
+You can run a permission check with permit.check(), passing in 3 arguments:
+
+- `user`: a unique string id or object that identifies the user performing the action.
+- `action`: the action performed.
+- `resource`: the resource (object) the action is performed on. Can be a string or an object.
+
+:::tip
```

Comment:
If we mentioning ABAC in this way, then we should also mention the ReBAC convention using the tuple convention of `:`

Reply:
I personally think it could be a much smaller comment about the argument types and do not necessarily mention ABAC/ReBAC

Reply:
This is an opinionated comment and I personally disagree, when giving examples of other models implementations, if we mention one model, I don't think there is necessary a need to always mention all the other models. ReBAC is a far more complex policy model to understand, and it's not relevant to be mentioning it's implementation in the quickstart. 

Initially you wanted me to mention ABAC and how the check function can change and be different, now you don't want me to mention ABAC. Can you give me an example how you want me to talk about mentioning other argument types (which is ABAC) without a reference to ABAC?

Reply:
I didn't express myself well in the first comment. I'll try again then.

The check function is not supposed to explain RBAC, ABAC, or ReBAC. The only thing we could explain is the different possible arguments. In our check function, there are three types of arguments—string for type, string for instance reference, and object (that could include some data).

I read my comment again and understood that I forgot to put an `etc.` after the example of the user as an object.  I didn't mean to mention ABAC in the first comment.

What I'm asking is that you mention briefly that the check function could have other arguments and refer to the main `check` page documentation.